setup.cfg

```diff
@@ -35,21 +35,16 @@ include = dbt, dbt.*
 
 [options.extras_require]
 dev =
-    black>=22.3.0
-    flake8>=4.0
-    isort>=5.10
-    mypy>=0.910
     pre-commit==2.15.0
     pytest==6.2.5
     pytest-dbt-adapter==0.6.0
 
-[mypy]
-disallow_untyped_defs = True
-show_error_codes = True
-
```

Comment:
I'd keep those and try to integrate mypy similar to how we have it in the SDK.

Reply:
I had taken this out because I was getting a weird error. I don't remember if it was local or on github. It's back in, and also in .pre-commit-config.yml.

Reply:
I'm moving mypy into a separate branch, bc it's asking for so many changes. I'm merging this for now; I don't know how long it'll take me to fix the 44 complaints mypy had, mostly because I'm too busy doing other crap.